scripts/microshift-cleanup-data.sh

```diff
@@ -116,17 +119,25 @@ if ${FULL_CLEAN} ; then
     echo "DATA LOSS WARNING: Do you wish to stop and clean ALL MicroShift data AND cri-o container workloads?"
     select yn in "Yes" "No"; do
         case "${yn}" in
-        Yes) 
+        Yes)
             break
             ;;
         *)
-            echo "Aborting cleanup" 
+            echo "Aborting cleanup"
             exit 0
             ;;
         esac
     done
 fi
 
+# Get the list of pod namespaces excluding 'openshift-ovn-kubernetes'
+ns_list=$(crictl pods -o json | jq -r '.items[].metadata.namespace' | sort | uniq | grep -v '^openshift-ovn-kubernetes$')
+# The 'openshift-ovn-kubernetes' namespace must be in the end of the list
+# to allow for pod deletion when MicroShift is stopped
+for ns in ${ns_list} openshift-ovn-kubernetes ; do
+    PODS_NS_LIST+=("${ns}")
+done
```

Comment:
Namespace list is computed dynamically now, including whatever namespaces user created.

Reply:
Why delete user workloads?

Reply:
We're deleting OVN pods, so networking will break. I thought it might be good to clean up user pods as well.
@dhellmann , if you think it's best not to touch those, I can revert to a hard-coded list like previously.